nservicebus/azure-service-bus/configuration/configuration.md

```diff
@@ -0,0 +1,200 @@
+ * `EnableExpress(Func<string, bool>, bool)`: Enables express mode when the given condition is true.
+ * `EnableFilteringMessagesBeforePublishing(bool)`: Enables message filtering before they are published, defaults to false.
+ * `EnablePartitioning(bool)`: Enables entity partitioning, defaults to false.
+ * `RequiresDuplicateDetection(bool)`: Specifies whether the topic should perform duplicate detection, defaults to false.
+ * `DuplicateDetectionHistoryTimeWindow(TimeSpan)`: The time period in which duplicate detection should occur. 
+ * `SupportOrdering(bool)`: Best effort message ordering on the topic, defaults to false.
+ * `DescriptionFactory(Func<string, ReadOnlySettings, TopicDescription>)`: A factory method that allows to create a TopicDescription object from the Azure Service Bus SDK. Use this factory method to override any (future) setting that is not supported by the Topics API.
+
+### Subscriptions
+
+The following settings are available to define how subscriptions should be created.
+ 
+ * `DefaultMessageTimeToLive(TimeSpan)`: The maximum age of a message, defaults to TimeSpan.MaxValue. 
+ * `EnableBatchedOperations(bool)`: Enables server side batched operations, defaults to true.
+ * `EnableDeadLetteringOnFilterEvaluationExceptions(bool)`: Dead letters messages when a filter evaluation doesn't match, defaults to false.
+ * `EnableDeadLetteringOnMessageExpiration(bool)`: Dead letters messages when they expire.
+ * `ForwardDeadLetteredMessagesTo(string)`: Forward all dead lettered message to the specified entity.
+ * `ForwardDeadLetteredMessagesTo(Func<string, bool>, string)`: Forward all dead lettered message to the specified entity if the given condition is true.
+ * `LockDuration(TimeSpan)`: The period of time that Azure Service Bus will lock a message before trying to redeliver it, defaults to 30 seconds.
+ * `MaxDeliveryCount(int)`: Sets the maximum delivery count, defaults to 10.
+ * `AutoDeleteOnIdle(TimeSpan)`: Auto deletes the entity if it hasn't been used for the given time frame.
+ * `DescriptionFactory(Func<string, string, ReadOnlySettings, SubscriptionDescription>)`: A factory method that allows to create a SubscriptionDescription object from the Azure Service Bus SDK. Use this factory method to override any (future) setting that is not supported by the Subscription API.
+ 
+## Controlling Connectivity
+
+The following settings determine how NServiceBus will connect to Azure Service Bus.
+ 
+ * `NumberOfClientsPerEntity(int)`: NServiceBus maintains a small pool of receive and send clients for each entity, this setting determines how big that pool is. Defaults to 5.
+ * `ConnectivityMode(ConnectivityMode)`: Determines how NServiceBus connects to Azure Service Bus, using TCP or HTTP. Defaults to TCP. 
+ * `BrokeredMessageBodyType(SupportedBrokeredMessageBodyTypes)`: Controls how the body of a brokered will be serialized as a byte array or as a stream. Defaults to byte array.
```

Comment:
> the body of a brokered will be

brokered **message** ?

> will be serialized as a

"will be serialized, as a" could also be "will be serialized, either as a"